[RHACS] Updated release notes for 4.4.2 patch release (#75827)
openshift/openshift-docs: release_notes/44-release-notes.adoc

*Release date*: 13 May 2024

This release provides the following bug fixes:

* Before this update, Collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.
* This release updates the Scanner baseline vulnerability data to address changes made to the Red{nbsp}Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were incorrectly indicated as affected by a vulnerability.
* This release fixes a crash and rendering error in the network graph that occurs when Central is running an {product-title-short} release of 4.3.6 or earlier and Sensor is running an {product-title-short} release of 4.4.0 or later.

This release provides the following change:

* The default telemetry endpoint is now set to a Red{nbsp}Hat proxy.

This releases updates the following items to patch vulnerabilities:

* Go has been updated to release 1.20.12.
* The `golang.org/x/net` module has been updated from release v0.22.0 to v0.23.0.

> 

Author: **Note:** This is the only modified part. Rest of the content is already reviewed and published in `rhacs-docs-4.4` branch but missing from the `rhacs-docs-main` branch.

+
This update ensures that the linked collection is retained, avoiding the error message.

[id="resolved-in-version-441_{context}"]
=== Resolved in version 4.4.1

*Release date*: 22 April 2024

* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.

[id="resolved-in-version-442_{context}"]
=== Resolved in version 4.4.2

*Release date*: 15 May 2024

This release provides the following bug fixes:

> * Before this update, Collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.

Reviewer: ```suggestion
* Before this update, collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.
```

> * This release updates the Scanner baseline vulnerability data to address changes made to the Red{nbsp}Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were incorrectly indicated as affected by a vulnerability.

Reviewer: ```suggestion
* This release updates the scanner baseline vulnerability data to address changes made to the Red{nbsp}Hat security data feeds that were not compatible with earlier data from scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were incorrectly indicated as affected by a vulnerability.
```

> * This release fixes a crash and rendering error in the network graph that occurs when Central is running an {product-title-short} release of 4.3.6 or earlier and Sensor is running an {product-title-short} release of 4.4.0 or later.

Reviewer: Sensor and Central seem like proper nouns, but I couldn't find anywhere else in the docs that they were capitalized without other modifiers. For example, Prism Central. Noting this as something to check.

* This release includes a new environment variable, `ROX_API_TOKEN_FILE`, that you can use to pass your API's token file path to the `roxctl` CLI.
* In earlier {product-title-short} versions, {product-title-short} did not update the alerts when violations changed. This release fixes the issue, and {product-title-short} correctly updates the alerts when violations change.

This release provides the following change:

* The default telemetry endpoint is now set to a Red{nbsp}Hat proxy.

This releases updates the following items to patch vulnerabilities:

* Go has been updated to release 1.20.12.

> * The `golang.org/x/net` module has been updated from release v0.22.0 to v0.23.0.

Author: Note: This is the only modified part. Rest of the content is already reviewed and published in rhacs-docs-4.4 branch but missing from the rhacs-docs-main branch.

Reviewer: Saw your note but was unclear on what this referred to. If I reviewed things outside the scope of the PR, my apologies.

[id="known-issues-440_{context}"]
== Known issues

*Updated on*: 22 April 2024

> * Currently, there is an issue for OpenShift Container Platform 4.12 on the {ibm-power} architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected.

Reviewer: ```suggestion
* Currently, there is an issue for {product-title} 4.12 on the {ibm-power} architecture where the CORE_BPF collection method does not work properly. From {product-title-short} 4.4.1 and later, Collector automatically uses the EBPF collection method in this particular case when CORE_BPF is selected.
```

Reviewer: Same as other note on `Collector` vs `collector`

> If you are using OpenShift Container Platform 4.12 on the {ibm-power} architecture and in offline mode, you must use a support package.

Reviewer: ```suggestion
If you are using {product-title} 4.12 on the {ibm-power} architecture and in offline mode, you must use a support package.
```
